.github/stale.yml

```diff
@@ -0,0 +1,9 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+daysUntilStale: 180
+daysUntilClose: false
```

Comment:
Let's also auto-close issues older than (say) 1 year.

Reply:
That would mean closing a lot of issues, which are definitely still valid, e.g. the two oldest open issues #8 and #13.

Reply:
Sure, but only if nobody comments on them to keep them open.

Reply:
Done.

I think what was not said is that even when an issue gets closed, it can be reopened if there is enough interest. Currently it is obvious that there is a management problem, since at the time of writing this we have 1.3k issues and 203 PRs open.

At the same time I'm not trying to say that by closing the issues those issues go away. Far from it. Being closed (and with a stale label) it means there was not enough interest to solve them.

And what happens is that this stale bot in a lot of cases serve as a reminder to get back to the issues/PRs. I know this happened to me on quite few occasions.

Reply:
The auto-closing is starting to kill contributions that I think shouldn't be killed:

* https://github.com/NixOS/nix/pull/3208#issuecomment-1100630134
* https://github.com/NixOS/nix/pull/3942#issuecomment-1100514524

Reply:
Iirc the open PR count had been hovering between 291 and 296 recently

Reply:
Closing was a terrible idea, and done in contra to the community's decision for Nixpkgs in https://github.com/NixOS/rfcs/blob/master/rfcs/0051-mark-stale-issues.md

Reply:
I have opened an RFC for this at https://github.com/NixOS/rfcs/pull/124